### ISSUE TITLE
Lack of default Sidecar configuration by operating system

### ISSUE THREAD

**Issue by PenguinSSH** [Graylog2/collector-sidecar#392]
### Problem description
Previous Graylog Sidecar Collector (Legacy) could define tags on the guest side.  We used this functionality to deploy a default tag to all managed endpoints so that a default configuration was applied.  

The current implementation, unless I'm mistaken, has no such functionality.  Scaling into the thousands of nodes, automatically applying tags to a group of nodes by OS, version, or type of name is critical to some operations.  Right now, this is a manual process prone to mistakes and automatic log collection for newly provisioned host is not possible.  

Please re-implement.  We are not able to migrate to the new version because of this drawback.  

### Steps to reproduce the problem

1. Install Graylog SideCar 1.0.2 according to documentation

### Environment
Windows 
* Sidecar Version: 1.0.2
* Graylog Version: 3.2.4
* Operating System: CentOS 7
* Elasticsearch Version: 6
* MongoDB Version:

**Comment by MiddleGeorgiaState** [comment]
Please bring back tags. We apply tags on installation to designate logs by department. Tags were also nice for pulling configurations based on OS and function.

**Comment by boosty** [comment]
Related: https://github.com/Graylog2/graylog2-server/issues/5062

**Comment by boosty** [comment]
Hi everyone,

with Graylog 5.0 we released the possibility to automatically assign Sidecar configurations, based on tags.

A list of tags can be assigned in a Sidecar's YAML configuration. Collector configuration matching any of these tags will then automatically be applied to the Sidecar.

Details:
- https://github.com/Graylog2/graylog2-server/pull/13433
- https://go2docs.graylog.org/5-0/getting_in_log_data/graylog_sidecar.html?Highlight=sidecar#AssigningTags

Recently a related change was added to the Sidecar installer on Windows to accept tags at installation time, for convenience:
- https://github.com/Graylog2/collector-sidecar/pull/464
